source/index-management/index-management.md

```diff
@@ -664,6 +664,15 @@ interface IndexOptions {
    * @example For an index of name: 1, age: -1, the generated name would be "name_1_age_-1".
    */
   name: String;
+  
+  /**
+   * Optionally specify a type for the index. Defaults to "search" if not provided.
```

Comment:
Many other driver options are implemented such that we only send a value if the user explicitly provides one. How do you feel about mandating that driver MUST NOT send this field if not provided?

Reply:
Makes sense to me if that's a consistent pattern among drivers. The server should default to `search` implicitly if `type` is not provided, so the behavior is consistent.

Reply:
`CreateSearchIndexOptions` is currently unused. If drivers support adding `type` to `IndexOptions`, that seems to be a better fit as then the top-level fields of`type` and `name` are present in the same interface.

Reply:
Actually, looking more closely it probably belongs in the `SearchIndexModel`?  

`IndexOptions` is used for regular indexes, not search indexes.

Reply:
Oh good catch! Yes, `SearchIndexModel` seems like the correct place for `type`.